src/api.js

```diff
@@ -163,16 +165,19 @@ api.declare({
       msg.organization = sanitizeGitHubField(body.repository.owner.login),
       msg.action = body.action;
       msg.details = getPullRequestDetails(body);
+      msg.installationID = body.installation.id;
```

Comment:
I want to do the work at some point to reduce the number of moving parts in tc-gh. There's no need for the indirection of having pulse in the middle and I'm 99% sure we got a random slowdown last week from having it there. That said, let's not do this work now!